common/app/common/configuration.scala

```diff
@@ -552,8 +552,6 @@ class GuardianConfiguration extends Logging {
     def mandatoryCredentials: AWSCredentialsProvider = credentials.getOrElse(throw new BadConfigurationException("AWS credentials are not configured"))
     val credentials: Option[AWSCredentialsProvider] = {
       val provider = new AWSCredentialsProviderChain(
-        new EnvironmentVariableCredentialsProvider(),
-        new SystemPropertiesCredentialsProvider(),
```

Comment:
I think the answer *is* yes :smile: 

We definitely want people using short-lived Janus credentials on their dev machines, and they are going to be in the AWS profile, not env variables. While re-ordering would stop the confusing behaviour I saw, I think it's better to remove `EnvironmentVariableCredentialsProvider` altogether - I don't think anyone's using it, and if they _are_, I want them using Janus profile credentials instead!

cc @adamnfish